test/test-parser.js

```diff
@@ -1419,6 +1419,181 @@ describe('errors', () => {
 
   it('throws for declarative shadow roots', () => {
     const callback = () => html`<template shadowrootmode="open"></template>`;
+    const expectedMessage = '[#156]';
+    assertThrows(callback, expectedMessage, { startsWith: true });
+  });
+});
+
+// Sanity check to make sure we’re able to hit all our errors.
+describe('errors coverage', () => {
```

Comment:
Added a dedicated `errors coverage` test section where we can add odds-and-ends tests which are required to meet our 100% coverage goals.